specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingProperty.json

```diff
@@ -1,62 +1,92 @@
 {
   "swagger": "2.0",
   "info": {
-    "description": "Billing client provides access to billing resources for Azure subscriptions.",
     "title": "BillingManagementClient",
+    "description": "Documentation for Microsoft.Billing.",
+    "contact": {
+      "name": "Microsoft.Billing",
+      "url": "https://learn.microsoft.com/en-us/rest/api/billing/",
+      "email": "cabbpt@microsoft.com"
+    },
     "version": "2023-04-01"
   },
-  "host": "management.azure.com",
-  "schemes": [
-    "https"
-  ],
-  "consumes": [
-    "application/json"
-  ],
-  "produces": [
-    "application/json"
-  ],
   "paths": {
     "/subscriptions/{subscriptionId}/providers/Microsoft.Billing/billingProperty/default": {
       "get": {
-        "description": "Get the billing properties for a subscription. This operation is not supported for billing accounts with agreement type Enterprise Agreement.",
+        "tags": [
+          "BillingProperty"
+        ],
+        "description": "Gets the billing properties for a subscription",
         "externalDocs": {
           "url": "https://docs.microsoft.com/en-us/rest/api/billing/"
         },
         "operationId": "BillingProperty_Get",
+        "produces": [
+          "application/json"
+        ],
         "parameters": [
           {
             "$ref": "./types.json#/parameters/subscriptionId"
           },
+          {
+            "in": "query",
+            "name": "includeBillingCountry",
```

Comment:
Is there a way we can make this a bit more general so we don't need to keep introducing flags per property? Can we categorize these secondary properties, and have "include<CategoryX>Properties"?

Reply:
@ramkd12 @amberbhargava to comment here